parent/pom.xml

```diff
@@ -56,10 +56,24 @@
         <wss4j.osgi.export>org.apache.wss4j.*</wss4j.osgi.export>
         <wss4j.osgi.dynamic.import />
         <wss4j.manifest.location>${project.build.outputDirectory}/META-INF/MANIFEST.MF</wss4j.manifest.location>
+        <jakarta.xml.bind-api.version>3.0.1</jakarta.xml.bind-api.version>
+        <jakarta.mail.version>2.0.0</jakarta.mail.version>
+        <jaxb-runtime.version>3.0.2</jaxb-runtime.version>
```

Comment:
There are a bunch of old JAXB dependencies in the root pom under the JDK9 profile that need to be removed:

<dependency>
                     <groupId>jakarta.xml.ws</groupId>
                     <artifactId>jakarta.xml.ws-api</artifactId>
                     <version>2.3.3</version>
                 </dependency>
                 <dependency>
                     <groupId>jakarta.xml.bind</groupId>
                     <artifactId>jakarta.xml.bind-api</artifactId>
                     <version>2.3.3</version>
                 </dependency>
                 <dependency>
                     <groupId>org.glassfish.jaxb</groupId>
                     <artifactId>jaxb-runtime</artifactId>
                     <version>2.3.2</version>
                     <scope>test</scope>
                 </dependency>
Once these are removed, then you can remove the jakarta API from ws-security-stax etc., as it is added already as a compile time dependency in Santuario (I changed it this morning so you might need to reinstall it).